.circleci/config.yml

```diff
@@ -170,7 +237,7 @@ workflows:
               only:
                 - /.*/
             tags:
-              only:
+              ignore:
                 - /v.*/
```

Comment:
should this be `/.*/`?  i assume we don't want to run `lint` for any tagging?  or do we want to run it for the non `/v.*/` tags?

Reply:
Yeah, this change was intentional. No need to run linting for version tags.